domains/Accounts/Controllers/AccountsLogoutController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Domains\Accounts\Controllers;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+class AccountsLogoutController extends Controller
+{
+    public function __invoke(Request $request): Response
+    {
+        $token = $request->user()->token();
+
+        $token->revoke();
+
+        return new Response('', Response::HTTP_OK);
```

Comment:
Nit: the 200 OK is, well, ok, but the 204 is a more suitable response code for when the call was processed and there's no actual response body to process. What do you think? :thinking:

Reply:
Yes, i thinks is better.